src/metrics.ts

```diff
@@ -0,0 +1,18 @@
+import { StatsD } from 'node-statsd'
+
+const environment = process.env.NODE_ENV
+const graphite = '127.0.0.1'
```

Comment:
Can we please configure this with an environment variable?  Preferably named GRAPHITE_HOST?